src/models/test_case_hook_definition_spec.ts

```diff
@@ -10,6 +10,11 @@ describe('TestCaseHookDefinition', () => {
         // Arrange
         const pickle = await getPickleWithTags([])
         const testCaseHookDefinition = new TestCaseHookDefinition({
+          code: undefined,
+          id: '',
+          line: 0,
+          uri: '',
+
```

Comment:
please remove this blank line. Appears a few times in this file

Reply:
Done